app/script/components/list/conversationListCallingCell.js

```diff
@@ -143,7 +143,14 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
     this.showParticipants = ko.observable(false);
 
     this.showVideoPreview = ko.pureComputed(() => {
-      return this.isVideoCall() && (this.multitasking.isMinimized() || !this.callIsConnected());
+      const hasOtherOngoingCalls =
+        this.calls()
+          .filter(call => call.id !== this.call().id)
+          .filter(call => call.state() === z.calling.enum.CALL_STATE.ONGOING).length > 0;
+
+      return (
+        !hasOtherOngoingCalls && this.isVideoCall() && (this.multitasking.isMinimized() || !this.callIsConnected())
+      );
```

Comment:
May I ask you to pull the last `or` bit out into a separats variable and use that in the return. I suppose the return fits into a single line then.

Reply:
There you go :)